src/ru/skypro/Main.java

```diff
@@ -1,8 +1,61 @@
 package ru.skypro;
 
+import java.util.*;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        List<Integer> nums = new ArrayList<>(List.of(1, 1, 2, 3, 4, 4, 5, 5, 6, 7));
+        List<String> quote = new ArrayList<>(List.of("easy", "come", "easy", "go"));
+        System.out.println("Odd numbers: " + getOddNums(nums));
+        System.out.println("Even numbers without dublicates: " + getUniqueEvenNums(nums));
+        System.out.println("Changed quote without duplicated words: " + getUniqueWords(quote));
+        System.out.println("Found " + countDuplicates(quote) + " duplicates.");
+    }
+
+    public static List getOddNums(List<Integer> nums) {
+        List<Integer> oddNums = new ArrayList<>();
+        for (int i = 0; i < nums.size(); i++) {
+            if (nums.get(i) % 2 != 0) {
+                oddNums.add(nums.get(i));
+            }
+        }
+        return oddNums;
+    }
+
+    public static List getUniqueEvenNums(List<Integer> nums) {
+        Set<Integer> set = new LinkedHashSet<>(nums);
+        nums.clear();
+        nums.addAll(set);
```

Comment:
В тот раз видимо не заметил, лучше не изменять исходный массив, который был передан в метод. Так как уже имеется уникальная коллекция в set, можно пройтись по ней и получить четные числа
```suggestion
       Set<Integer> set = new LinkedHashSet<>(nums);

        List<Integer> evenNums = new ArrayList<>();

        for (Integer num : set) {
            if (num % 2 == 0) {
                evenNums.add(num);
            }
        }
        return evenNums;
```

Reply:
Также и с методом getUniqueWords